.github/workflows/prod.yml

```diff
@@ -0,0 +1,51 @@
+name: Production
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  laravel-tests:
+    runs-on: self-hosted
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Copy .env
+      run: php -r "file_exists('.env')
+      working-directory: ./src
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist --optimize-autoloader --no-dev
+      working-directory: ./src
+    - name: Generate key
+      run: php artisan key:generate
+      working-directory: ./src
+    - name: Enable Artisan in prod mode
+      run: php artisan config:cache && php artisan route:cache
+    - name: Create Database
+      run: |
+        mkdir -p database
+        touch database/database.sqlite
+      working-directory: ./src
+    - name: Execute tests (Unit and Feature tests) via PHPUnit
+      env:
+        DB_CONNECTION: sqlite
```

Comment:
Change to postgres

Reply:
Forgot to take changes from Testing pipeline, fixed in f7e1199ccf4a51fdbfc14bff54a54f9e4515872e